frontend/src/stores/userStore.ts

```diff
@@ -17,12 +18,10 @@ export const useUserStore = defineStore("users", {
   },
   // actions == methods
   actions: {
-    async login() {
-      const loginData = {
-        name: "renoster",
-          password: 'R'
-        }
-        await postRequest("login", loginData);
+    async login(loginForm: LoginForm) {
+      await postRequest("login", loginForm);
+      await this.refreshMe();
+      await router.push("/")
```

Comment:
If the user inputs wrong info, or nothing, there is an error in the console, but nothing else happens.
Do you want catching this error to be a separate ticket and handled later, or do you want to implement that now?

Reply:
it would also change something in the html btw, since you'd want to show an error message visually. So maybe a separate task is nice.. but you pick!

Reply:
Could also maybe add an options, that if a username is entered that exists as an intra_user we show a message "did you mean to login through the API?" or something like that..

Reply:
added a simple alert in the try catch block of userStore->login() and added a ticket #130